docs/application/native/guides/connectivity/vpn.md

```diff
@@ -30,7 +30,7 @@ The main features of the VPN Service API include:
   The VPN profile provides different functions for routing management, DNS management, and firewall management:
 
   - The `vpnsvc_init()` function is used to a initialize the VPN interface and get the handle.
-  - The `vpnsvc_h` handle is used to [configure the tunnel interface](#config) before using the [VPN application control](../app-management/common-appcontrol.md#vpnservice) to ask the user permission to connect to the VPN service.
+  - The `vpnsvc_h` handle is used to [configure the tunnel interface](#config) before using the [VPN application control](../app-management/common-appcontrols.md#vpnservice) to ask the user permission to connect to the VPN service.
```

Comment:
It looks like the \<a name\> tags are removed, and we are using links generated by \# only.
`vpnservice` is old tag and removed now. To jump to the right location, it should be `vpn-service`.